### PR TITLE
Update developer-guide.md

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: documentation/project-docs/developer-guide.md [hunk 1]
```diff
@@ -101,6 +101,13 @@ This should output `.../artifacts/tmp/Debug/dotnet/dotnet`.
 
 You can now run `dotnet` commands to test changes.
 
+## Run tests in commandline
```
Comment (Youssef1313) [inline documentation/project-docs/developer-guide.md:104]:
```suggestion
## Run tests in command line

```

File: documentation/project-docs/developer-guide.md [hunk 2]
```diff
@@ -101,6 +101,13 @@ This should output `.../artifacts/tmp/Debug/dotnet/dotnet`.
...
+```shell
+build.cmd # to have a full build first
+.\artifacts\sdk-build-env.bat
+cd src\Tests\YOURTEST.Tests # cd to the test folder that contains the test csproj file
+dotnet test --filter "FullyQualifiedName~TESTNAME" # run individual test
+```
 ## A simple test
```
Comment (Youssef1313) [inline documentation/project-docs/developer-guide.md:111]:
```suggestion

## A simple test
```

**Review by Youssef1313 — COMMENTED** [review]
Changed commandline to command-line, and fixed a violation of [MD031](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines) markdown rule.

File: documentation/project-docs/developer-guide.md [hunk 3]
```diff
@@ -101,6 +101,15 @@ This should output `.../artifacts/tmp/Debug/dotnet/dotnet`.
 
 You can now run `dotnet` commands to test changes.
 
+## Run tests in command line
```
Comment (dsplaisted) [inline documentation/project-docs/developer-guide.md:104]:
```suggestion
## Run tests from the command line
```

File: documentation/project-docs/developer-guide.md [hunk 4]
```diff
@@ -101,6 +101,15 @@ This should output `.../artifacts/tmp/Debug/dotnet/dotnet`.
...
+
+```shell
+build.cmd # to have a full build first
+.\artifacts\sdk-build-env.bat
+cd src\Tests\YOURTEST.Tests # cd to the test folder that contains the test csproj file
+dotnet test --filter "FullyQualifiedName~TESTNAME" # run individual test
```
Comment (dsplaisted) [inline documentation/project-docs/developer-guide.md:110]: I typically do something like `dotnet run -method NameSpace.Class.TestMethod` instead.  But I'm fine with doc'ing it this way.

**Review by dsplaisted — APPROVED** [review]

**Pull request merged**